engine/release-notes/index.md

```diff
@@ -13,896 +13,164 @@ redirect_from:
+- Added a new `docker context show` command that prints the name of the current context. [docker/cli#3567](https://github.com/docker/cli/pull/3567)
+- Added a `--format=json` option to `docker inspect` and `docker ls`. [docker/cli#2936](https://github.com/docker/cli/pull/2936)
+- Added a `--quiet` option to `docker create` and `docker run` commands to suppress output when pulling an image. [docker/cli#3377](https://github.com/docker/cli/pull/3377)
+- Added a `--force` option to `docker network rm` subcommand. Causes CLI to return a 0 exit code even if the network does not exist. Has no effect on the server-side procedure for removing a network. [docker/cli#3547](https://github.com/docker/cli/pull/3547)
+- Added a `--signal` option to `docker stop` and `docker restart`. [docker/cli#3614](https://github.com/docker/cli/pull/3614)
+- Allow use of `STOPSIGNAL` via `docker commit`. [moby/moby#43369](https://github.com/moby/moby/pull/43369)
+- Added a new option to the `awslogs` log driver to allow skipping log stream creation in CloudWatch. [moby/moby#42132](https://github.com/moby/moby/pull/42132)
+- Added a new option to the `awslogs` log driver to specify the log format that is sent to CloudWatch. [moby/moby#42838](https://github.com/moby/moby/pull/42838)
+- Added a new option to the `fluentd` log driver to set the reconnection interval. [moby/moby#43100](https://github.com/moby/moby/pull/43100)
+- Added new options-setters to the Go API client: `WithTLSClientConfigFromEnv()`, `WithHostFromEnv()`, and `WithVersionFromEnv()`. [moby/moby#42224](https://github.com/moby/moby/pull/42224)
+- Added generation of shell command completion through `docker completion` subcommand. [docker/cli#3429](https://github.com/docker/cli/pull/3429)
+- API: Added a `Swarm` header to `GET /_ping` and `HEAD /_ping`, allowing single-request detection of Swarm support. [moby/moby#42064](https://github.com/moby/moby/pull/42064)
+- API: Added a `signal` parameter to `POST /containers/{id}/stop` and `POST /containers/{id}/restart` to set the signal used. [moby/moby#43206](https://github.com/moby/moby/pull/43206)
+- API: Added a `CreateMountPoint` parameter to `POST /containers/create`. [moby/moby#43484](https://github.com/moby/moby/pull/43484)
+- API: Added a `shared-size` parameter to `GET /images/json` to enable shared-size computation of images. [moby/moby#42531](https://github.com/moby/moby/pull/42531)
+- API: Added a `type` parameter to `GET /system/df`, to control what object types to are considered when computing disk usage. [moby/moby#42559](https://github.com/moby/moby/pull/42559)
+- systemd: Start `docker.service` after `time-set.target`. [moby/moby#43107](https://github.com/moby/moby/pull/43107)
+
+### Removed
+
+- Removed support for reading configuration from `~/.dockercfg`. [docker/cli#2504](https://github.com/docker/cli/pull/2504)
```

Comment:
Similar to the API changes, we could provide a link to the https://docs.docker.com/engine/deprecated/ page for a full overview of deprecated (and/or removed) features in this release. Some thing may be marked deprecated (but not yet removed), and should no longer be used (even if they are currently still functional).

Reply:
added top-level links to both the API reference and deprecated pages